src/leo_nlm_lock_handler.erl

```diff
@@ -0,0 +1,366 @@
+%%======================================================================
+%%
+%% Network Lock Manager written in Erlang
```

Comment:
Add a description about how to realise this feature.
In this case, using ets and so on.